components/checkbox.module.css

```diff
@@ -0,0 +1,22 @@
+.checkbox {
+  position: relative;
+  height: 24px;
+  width: 24px;
+  border: 1px solid #d1d6dc;
+  border-radius: 4px;
+
+  &:after {
+    content: "";
+    position: absolute;
+    top: -5px;
+    left: -5px;
+    height: calc(100% + 10px);
```

Comment:
`calc`s might work in `className` as `’h-[calc(100% + 10px)] w-[calc(100% + 10px)]’—i dont actually remember on that one

why isnt this possible with negative margin though? `’after:(content absolute inset-0 -m-5)’` should give you the same box